teams_accounts.go

```diff
@@ -75,6 +75,7 @@ type TeamsBlockPage struct {
 	Name            string `json:"name,omitempty"`
 	MailtoAddress   string `json:"mailto_address,omitempty"`
 	MailtoSubject   string `json:"mailto_subject,omitempty"`
+	SuppressFooter  bool   `json:"suppress_footer,omitempty"`
```

Comment:
can we please make this a boolean pointer? i'm growing hesitant to use only booleans with API calls these due to the number of issues arising from it.

Reply:
done 👍